starmark-integration/tests/test-helpers.ts

```
@@ -35,4 +35,76 @@ export async function renderAstroComponentToString(
 ) {
   const container = await AstroContainer.create();
   return await container.renderToString(Component, options);
+}
+
+/**
+ * Mock Storage Connector for testing
+ */
+export class MockStorageConnector {
+  public name: string;
+  private detectionResult: boolean = true;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)